tock/employees/models.py

```diff
@@ -110,6 +110,10 @@ class Meta:
     def __str__(self):
         return '{0}'.format(self.user)
 
+    @property
+    def is_active(self):
```

Comment:
I wonder if there's a good argument for us to use `@cached_property` here, thinking this is relatively expensive to compute (requires a db traversal) and changes infrequently. 

I wonder about this with a lot of our properties, tbh.

https://docs.djangoproject.com/en/3.0/ref/utils/#django.utils.functional.cached_property